arches_search/migrations/0007_update_search_table_indices.py

```diff
@@ -0,0 +1,215 @@
+import django.contrib.postgres.indexes
+from django.contrib.postgres.operations import TrigramExtension
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("arches_search", "0006_add_more_specific_indices_to_search_tables"),
+    ]
+
+    operations = [
+        TrigramExtension(),
+        migrations.RemoveIndex(
+            model_name="booleansearch",
+            name="arches_sear_tileid_0b23d0_idx",
+        ),
+        migrations.RemoveIndex(
+            model_name="booleansearch",
+            name="arches_sear_graph_s_cdea05_idx",
+        ),
+        migrations.RemoveIndex(
+            model_name="booleansearch",
+            name="arches_sear_graph_s_f9da56_idx",
+        ),
```

Comment:
no need to remove this index once we just don't add it in 0006

Reply:
I'd disagree with you here @chiatt . I definitely think that these 2 migrations should collapsed, but I think merging _all_ migrations into one giant amalgam makes for harder maintenance